### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by ldez**
### Description

- add pseudo menu to readme.
- move maintainers list to `MAINTAINER.md`
- reorganize readme section.
- fix headers level in `CONTRIBUTING.md`
- add "How to" for issue and PR l in `CONTRIBUTING.md`
- update labels in `MAINTAINER.md`

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -173,3 +173,54 @@ INFO    -  Cleaning site directory
...
+
+- Make it small.
+- Do only one thing.
+- Write useful descriptions and titles.
+- Avoid re-formatting.
+- Make sure the code builds.
+- Make sure all tests pass.
+- Add tests.
+- Address review comments in terms of additional commits.
+- Do not amend/squash existing ones unless the PR is trivial.
```
Comment (dtomcej) [inline .github/CONTRIBUTING.md]: Does this mean that we are not wanting people to squash commits?

File: README.md [hunk 2]
```diff
@@ -142,17 +152,29 @@ If you prefer commercial support, please contact [containo.us](https://containo.
...
-- Julien Salleyron [@juliens](https://github.com/juliens)
-- Nicolas Mengin [@nmengin](https://github.com/nmengin)
+[Information about process and maintainers](.github/MAINTAINER.md)
+
+
+## Plumbing
+
+- [Oxy](https://github.com/vulcand/oxy): an awesome proxy library made by Mailgun folks
+- [Gorilla mux](https://github.com/gorilla/mux): famous request router
+- [Negroni](https://github.com/urfave/negroni): web middlewares made simple
```
Comment (dtomcej) [inline README.md]: Aren't we removing this?

Comment (ldez) [inline README.md]: We have an issue on this subject: https://github.com/containous/traefik/issues/1730

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (ldez) [inline .github/CONTRIBUTING.md]: During the review process only.

**Pull request merged**